src/Sentry.Unity/UnityEventProcessor.cs

```diff
@@ -273,6 +275,14 @@ private void PopulateTags(SentryEvent @event)
         }
     }
 
+
+    internal class NativeStackTrace
+    {
+        public IntPtr[] Frames = null!;
```

Comment:
Doesn't look safe to `null!` here. Start off with an instance, or make this nullable. 
```suggestion
        public IntPtr[] Frames = new IntPtr[0];
```

```suggestion
        public IntPtr[]? Frames { get; set; }
```